examples/webgl_loader_ifc.html

```diff
@@ -25,7 +25,11 @@
 			{
 				"imports": {
 					"three": "../build/three.module.js",
-					"three/addons/": "./jsm/"
+					"three/addons/": "./jsm/",
+					"three/examples/jsm/utils/BufferGeometryUtils": "./jsm/utils/BufferGeometryUtils.js",
+					"three-mesh-bvh": "https://unpkg.com/three-mesh-bvh@^0.5.21/build/index.module.js",
```

Comment:
In the other examples we used fixed releases to avoid inadvertant breakage from package updates.

Reply:
Also, is a import of `three-mesh-bvh` needed here?

Reply:
> Hey @gkjohnson are you suggesting to import a fixed version of three?

I should have been more specific - the `^` version specifier means the version of three-mesh-bvh is not fixed. It would be
best to just lock the version at the current latest without the caret. See https://github.com/mrdoob/three.js/pull/23907#discussion_r851722322.

Reply:
Got it, done!